leakcanary-analyzer/src/test/java/com/squareup/leakcanary/RetainedSizeTest.java

```diff
@@ -21,7 +21,9 @@
   @Parameterized.Parameters public static Collection<Object[]> data() {
     return Arrays.asList(new Object[][] {
         { ASYNC_TASK_PRE_M, 207_407 }, //
+        // TODO why did this change to 145353 with latest perflib?
```

Comment:
Before: Retained: 1870 and bitmap: 0
After: Retained: 49584 and bitmap: 95769
YourKit says retained size is 1920